gensrc/thrift/DataSinks.thrift

```diff
@@ -207,3 +207,22 @@ struct TDataSink {
   8: optional TMemoryScratchSink memory_scratch_sink
   9: optional TMultiCastDataStreamSink multi_cast_stream_sink
 }
+
+struct TIcebergColumnStats {
```

Comment:
If that iceberg table's desc? Maybe put it into `TIcebergTable` in `Descriptors.thrift`?

Reply:
This is the ColumnStats used in sink, Descriptors.thrift is about desc of table/column/slot/file and so on, DataSinks.thrift is more suitable, do you think so?